src/views/ProductDetail.vue

```diff
@@ -60,101 +60,74 @@
               <ion-list v-if="selectedSegment === 'inStore'">
                 <ion-item>
                   <ion-label class="ion-text-wrap">{{ translate("Quantity on hands")}}</ion-label>
-                  <ion-note slot="end">10</ion-note>
+                  <ion-note slot="end">{{ getProductStock(product.variants[0].productId)?.quantityOnHandTotal ?? '0' }}</ion-note>
                 </ion-item>
                 <ion-item>
                   <ion-label class="ion-text-wrap">{{ translate("Safety stock")}}</ion-label>
-                  <ion-note slot="end">10</ion-note>
+                  <ion-note slot="end">{{ getInventoryInformation(product.variants[0].productId)?.minimumStock ?? '0' }}</ion-note>
```

Comment:
Need to handle the case when variant is changed as we have currently hardcoded `variants[0]`, but we need to update the inventory based on color and size selected.